src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ContentTypeExtensions.cs

```diff
@@ -45,37 +43,43 @@ public static bool IsJson(this ContentType contentType)
 
             string acceptedMainType = "application";
             string acceptedSubType = "json";
-            string mediaType = contentType.MediaType;
-
-            if (!ExcludedJsonContentTypes.Contains(mediaType, StringComparer.OrdinalIgnoreCase))
-            {
-                ReadOnlySpan<char> mediaTypeSpan = mediaType.AsSpan();
 
-                // Since contentType has been validated using System.Net.Mime.ContentType,
-                // mediaType will always have exactly 2 parts after splitting on '/'
-                int slashIndex = mediaTypeSpan.IndexOf('/');
+            ReadOnlySpan<char> mediaTypeSpan = contentType.MediaType.AsSpan();
 
-                if (mediaTypeSpan.Slice(0, slashIndex).Equals(acceptedMainType.AsSpan(), StringComparison.OrdinalIgnoreCase))
-                {
-                    ReadOnlySpan<char> subTypeSpan = mediaTypeSpan.Slice(slashIndex + 1);
+            // Since contentType has been validated using System.Net.Mime.ContentType,
+            // mediaType will always have exactly 2 parts after splitting on '/'
+            int slashIndex = mediaTypeSpan.IndexOf('/');
 
-                    while (!subTypeSpan.IsEmpty)
-                    {
-                        int plusIndex = subTypeSpan.IndexOf('+');
+            if (mediaTypeSpan.Slice(0, slashIndex).Equals(acceptedMainType.AsSpan(), StringComparison.OrdinalIgnoreCase))
+            {
+                ReadOnlySpan<char> subTypeSpan = mediaTypeSpan.Slice(slashIndex + 1);
 
-                        ReadOnlySpan<char> currentSubType = plusIndex == -1 ? subTypeSpan : subTypeSpan.Slice(0, plusIndex);
+                while (!subTypeSpan.IsEmpty)
+                {
+                    int plusIndex = subTypeSpan.IndexOf('+');
 
-                        if (currentSubType.Equals(acceptedSubType.AsSpan(), StringComparison.OrdinalIgnoreCase))
-                        {
-                            return true;
-                        }
+                    ReadOnlySpan<char> currentSubType = plusIndex == -1 ? subTypeSpan : subTypeSpan.Slice(0, plusIndex);
 
-                        subTypeSpan = plusIndex == -1 ? ReadOnlySpan<char>.Empty : subTypeSpan.Slice(plusIndex + 1);
+                    if (currentSubType.Equals(acceptedSubType.AsSpan(), StringComparison.OrdinalIgnoreCase))
+                    {
+                        return true;
                     }
+
+                    subTypeSpan = plusIndex == -1 ? ReadOnlySpan<char>.Empty : subTypeSpan.Slice(plusIndex + 1);
                 }
             }
 
             return false;
         }
+
+        public static bool IsFeatureFlag(this ContentType contentType)
```

Comment:
Why don't the ff and kv adapters use these extensions?

Reply:
Good point, I didn't realize since we only used the `ContentType` class in the `JsonKeyValueAdapter`, but I can add it to those other classes

Reply:
If we're going to add it, I'd say we should.

Reply:
updated